internal/command/launch/launch_frameworks.go

```diff
@@ -128,7 +128,7 @@ func (state *launchState) scannerRunCallback(ctx context.Context) error {
 		return nil
 	}
 
-	err := state.sourceInfo.Callback(state.Plan.AppName, state.sourceInfo, state.Plan)
+	err := state.sourceInfo.Callback(state.Plan.AppName, state.sourceInfo, state.Plan, flag.ExtraArgsFromContext(ctx))
```

Comment:
Quick question! 
This new arg is passed to a framework-scanner callback right? 
Right now I can see that only 4 framework scanners' callbacks where updated to receive this new argument. Will other scanners need their callback functions to be updated to receive the new arg?